jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -464,6 +527,9 @@ def calculate_photometry(self, dataset=None, aperture=None, background=None,
                    'data_label', 'subset_label', 'timestamp'],
             indexes=[1, 1, 1, 1, 3, 3, 3, 3, 3, 3, 18, 18, 18])
 
+        if self.config == "cubeviz":

```

Comment:
and cube-like?

Reply:
Can't do that because if you do photometry on a cube, and then on its moment map, if moment map does not have that column, it won't append. And since it is QTable, it has to be an int.

Unless you want me to change that column into str and I leave it empty for moment map? Or you want me to stick a NaN into the column for moment map if the column stays an int type?

Reply:
I think NaN makes the most sense for now.  Although maybe this just further motivates moving away from QTable for these things in the future...

Reply:
`np.nan` shows up as weird int value in QTable. I will ponder more tomorrow.

Reply:
Implemented wavelength as Cami suggested. When 2D data is selected in Cubeviz, it would be NaN in the table.